src/__tests__/plugin-test.js

```diff
@@ -0,0 +1,43 @@
+import Microcosm  from '../Microcosm'
+
+describe('Microcosm', function() {
+
+  it ('can add plugins', function(done) {
+    let m = new Microcosm()
+
+    m.addPlugin({
+      register: function(app, options, next) {
+        next()
+      }
+    })
```

Comment:
I meant, you're not using `() =>` or `register(app, options, next)` signatures. I assumed it was because running babel on tests is a pain, but looks like I'm wrong.